_security/configuration/tls.md

```diff
@@ -264,15 +264,39 @@ These settings allow for the use of encrypted passwords in the settings.
 
 ## Hot reloading TLS certificates
 
-Updating expired or nearly expired TLS certificates does not require restarting the cluster. Instead, enable hot reloading of TLS cerificates by adding the following line to `opensearch.yml`:
+Updating expired or nearly expired TLS certificates on the HTTP and transport layers does not require restarting the cluster. Instead, you can enable hot reloading of TLS certificates using one of the following methods:
 
+- [Enabling the in-place hot reload setting](#enabling-the-in-place-hot-reload-setting)
+- Using the [Reload Certificates API](#using-the-reload-certificates-api)
 
-`plugins.security.ssl_cert_reload_enabled: true`
+If you use PEM files to manage certificates, your certificate and keys are stored in seperate files. Therefore, we recommend using rolling updates when updating hot reload settings to prevent authentication issues.
```

Comment:
```suggestion
If you use PEM files to manage certificates, your certificate and keys are stored in separate files. Therefore, we recommend using rolling updates when updating hot reload settings to prevent authentication issues.
```